src/bindgen/config.rs

```diff
@@ -293,6 +315,8 @@ pub struct FunctionConfig {
     pub rename_args: Option<RenameRule>,
     /// An optional macro to use when generating Swift function name attributes
     pub swift_name_macro: Option<String>,
+    /// Select sort stradegy for function names
+    pub sort_by: SortStradegy,
```

Comment:
You mean `strategy`, right? Not stradegy.

Reply:
oh dear! :see_no_evil: